Rakefile

```diff
@@ -22,18 +22,23 @@
 # ------------------------------------------------------------- Constants ------
 
 PROJECT_IOS = 'MobileVLCKit.xcodeproj'
+PROJECT_TV = 'MobileVLCKit.xcodeproj'
 PROJECT_MAC = 'VLCKit.xcodeproj'
 
 SDK_SIM_IOS = 'iphonesimulator11.3'
+SDK_SIM_TV = 'appletvsimulator11.3'
 SDK_SIM_MAC = 'macosx10.13'
 
 SDK_SIM_DEST_IOS = "'platform=iOS Simulator,name=iPhone 7,OS=11.3'"
+SDK_SIM_DEST_TV = "'platform=tvOS Simulator,name=Apple TV,OS=11.3'"
 SDK_SIM_DEST_MAC = "'platform=OS X,arch=x86_64'"
 
 SCHEME_IOS = 'MobileVLCKitTests'
+SCHEME_TV = 'TVVLCKitTests'
 SCHEME_MAC = 'VLCKitTests'
 
 VLC_FLAGS_IOS = '-dva x86_64'
+VLC_FLAGS_TV = '-ftb'
```

Comment:
would -stb be enough? we don't need it for a device here, do we?

Reply:
Seems like using `-stb` fails to compile 🤔https://circleci.com/gh/videolan/vlckit/42

Reply:
@Mikanbu fixed the underlying issue can we retry with stb here?